CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+TBD
+---
+
+- Add simple filtering if provided a minutes argument in `clean_duplicate_history`
```

Comment:
can you add `(gh-606)` to the end of this?

Reply:
Added.